docs/medical-api/getting-started/quickstart.mdx

```diff
@@ -219,3 +219,17 @@ That's it! You're now well equiped to use the Metriport API to access and manage
 
 As next steps, we'd recommend scoping out our [Medical API Reference](/medical-api/api-reference/organization/create-organization)
 in further detail to see the other things you can do with the API.
+
+## 8. Get Patient Resources
+
+As we query for documents we will automatically convert CDA docs into FHIR.
+These C-CDA documents can come in multiple formats such as:
+
+- XML
+- PDFs
+- Images
```

Comment:
"These C-CDA documents can come in multiple formats" -> this isn't accurate. Perhaps:

```
As we query for Documents we will automatically convert XML C-CDA docs into FHIR resources.

<Tip>
   Other Documents, like PDFs and images, will be left in their original format and not processed.
</Tip>
```